src/utils/pdfExport.ts

```
@@ -0,0 +1,765 @@
+            opacity: styles.opacity,
+            className: el.className,
+            text: el.textContent?.substring(0, 50)
+          });
+        });
+      });
+      
+      // Check all elements with "legend" in class name
+      const allElements = fullChartContainer.querySelectorAll('*');
+      const legendRelated = Array.from(allElements).filter(el => 
+        el.className && el.className.toString().toLowerCase().includes('legend')
+      );
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)